src/status_im/data_store/invitations.cljs

```diff
@@ -0,0 +1,8 @@
+(ns status-im.data-store.invitations
+  (:require clojure.set))
+
+(defn <-rpc [message]
```

Comment:
Destructuring is much faster, and probably just as readable
```
(def a {:chatId "a"  :introductionMessage "message" :messageType "message-type"})

(defn <-rpc-rename [message]
  (-> message
      (clojure.set/rename-keys {:chatId              :chat-id
                                :introductionMessage :introduction-message
                                :messageType         :message-type})))

(defn <-rpc-destruct [{:keys [chatId introductionMessage messageType]}]
  {:chat-id chatId
   :introduction-message introductionMessage
   :message-type messageType})


(defn time-rename []
  (time
   (doseq [_ (range 10000)]
     (<-rpc-rename a))))

(defn time-destruct []
  (time
   (doseq [_ (range 10000)]
     (<-rpc-destruct a))))

cljs.user=> (time-rename)
"Elapsed time: 68.831563 msecs"
nil
cljs.user=> (time-destruct)
"Elapsed time: 14.788217 msecs"

```
destructring is around 4/5 times faster, probably does not matter much, but I would just use destructuring here.

Reply:
thanks , wondering why

Reply:
thanks

Reply:
but you create a new map, but i need to remove keys and assign new keys

Reply:
also this approach is general for the app, not just introduced in this PR

Reply:
up to you, it's true that we use rename-keys more often, maybe it's time to move away from it :), but either are fine with me